servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -174,62 +160,59 @@ def _process_components(self, components, instance_config):
                 identifiers.append("urn:host:/{}".format(comp_name))
             identifiers.append(external_id)
             data.update(component)
-            data.update({"identifiers": identifiers, "tags": instance_config.instance_tags})
+            data.update({"identifiers": identifiers, "tags": instance_info.instance_tags})
```

Comment:
I just noticed we still have this part here:
```
if 'fqdn' in component and component['fqdn']:
    identifiers.append("urn:host:/{}".format(component['fqdn']))	                
if 'host_name' in component and component['host_name']:
    identifiers.append("urn:host:/{}".format(component['host_name']))	               
else:
    identifiers.append("urn:host:/{}".format(comp_name))	             
```

Can you convert it to use the identifier functions like Dynatrace does: 
https://github.com/StackVista/stackstate-agent-integrations/blob/master/dynatrace_topology/stackstate_checks/dynatrace_topology/dynatrace_topology.py#L5

https://github.com/StackVista/stackstate-agent-integrations/blob/master/dynatrace_topology/stackstate_checks/dynatrace_topology/dynatrace_topology.py#L184

Reply:
Done.